### Hi there, I'm Lei Li

## I'm a Lead Bioinformatics Research Scientist at St. Jude Children's Research Hospital.
I am interested in developing computational tools and algorithms to facilitate biological researches.

- Read more about my [Personal page](https://leili-uchicago.github.io/)
- Read more about my [Research](https://scholar.google.com/citations?user=3qt2VoAAAAAJ&hl=en)
- Reach me out by [Personal Email](mailto:leilioxford@gmail.com) or [Work email](mailto:lei.li@stjude.org)

<a href="https://leili-uchicago.github.io/">
    <img src="https://github-readme-stats.vercel.app/api?username=leili-uchicago&show_icons=true&icon_color=0366d6&text_color=24292e&bg_color=ffffff&count_private=true" />
    <img src="https://github-profile-trophy.vercel.app/?username=leili-uchicago&theme=flat&title=Stars,Followers,Commit,MultiLanguage&margin-w=5&row=1&column=4" />
</a>

</br>

## RECENT PROJECTS
<img src="https://github.com/stjudecab/RSVreconPy/blob/main/Resource/RSV_icon.png" align="left" height="80px" width="80px" style="margin-right: 20px; margin-bottom: 20px;">  [RSVrecon](https://github.com/stjudecab/RSVreconPy): An efficient workflow to screen clinically relevant genomic variations of RSV from NGS data.<br> &nbsp; &nbsp; <br> &nbsp; &nbsp; <br> <br> 

<img src="https://leili-uchicago.github.io/img/Htoreader_icon.png" align="left" height="80px" width="80px" style="margin-right: 20px; margin-bottom: 20px;"> A hybrid demultiplexing strategy that improves performance and robustness of cell hashing. [Paper](https://academic.oup.com/bib/article/25/4/bbae254/7686601?login=false) [Github](https://github.com/WilsonImmunologyLab/HTOreader) <br> &nbsp; &nbsp; <br> &nbsp; &nbsp; <br> <br> 

<img src="https://wilsonimmunologylab.github.io/VGenes/img/logo.png" align="left" height="80px" width="80px" style="margin-right: 20px; margin-bottom: 20px;">  [VGenes](https://wilsonimmunologylab.github.io/VGenes/): an integrated graphical tool for efficient, comprehensive and multimodal analyses of massive B-cell repertoire sequences.<br> &nbsp; &nbsp; <br> &nbsp; &nbsp; <br> <br> 

<img src="https://wilsonimmunologylab.github.io/Librator/img/logo.png" align="left" height="80px" width="80px" style="margin-right: 20px; margin-bottom: 20px;">  [Librator](https://wilsonimmunologylab.github.io/Librator/): a platform for optimized sequence editing, design, and expression of influenza virus proteins.<br> &nbsp; &nbsp; <br> &nbsp; &nbsp; <br> <br> 

<img src="https://wilsonimmunologylab.github.io/LinQView/img/linklogo.png" align="left" height="80px" width="80px" style="margin-right: 20px; margin-bottom: 20px;">  [LinQ-View](https://wilsonimmunologylab.github.io/LinQView/) is a joint single cell analysis strategy that could integrate information from both transcriptome and surface protein markers for cell heterogeneity identification. <br> &nbsp; &nbsp;  <br> <br> 

<img src="https://wilsonimmunologylab.github.io/Cookie/img/logo.png" align="left" height="80px" width="80px" style="margin-right: 20px; margin-bottom: 20px;">  [Cookie](https://wilsonimmunologylab.github.io/Cookie/): Selecting representative samples from single cell atlas using k-medoids clustering.<br> &nbsp; &nbsp; <br> &nbsp; &nbsp; <br> 
